lib/ledgerEventStorage.js

```diff
@@ -267,6 +264,31 @@ module.exports = class LedgerEventStorage {
     });
   }
 
+  getActiveConfig({blockHeight}, callback) {
+    assert.optionalNumber(blockHeight, 'blockHeight');
+    const query = {
+      'event.type': 'WebLedgerConfigurationEvent',
+      'meta.blockHeight': {$lt: blockHeight},
```

Comment:
Maybe add a note here that says that an "active" config does not include any configs that may be in the current block -- so just it's clear this `$lt` is not a mistake to readers in the future.